egs/wham/ConvTasNet/local/prepare_data.sh

```diff
@@ -11,6 +11,7 @@ mkdir -p $out_dir
 echo "Download WHAM noises into $out_dir"
 # If downloading stalls for more than 20s, relaunch from previous state.
 wget -c --tries=0 --read-timeout=20 https://storage.googleapis.com/whisper-public/wham_noise.zip -P $out_dir
+unzip $out_dir/wham_noise.zip -d $out_dir
```

Comment:
Nice catch, can we redirect it in the logs  with this `>> logs/unzip_wham.log` please?

Reply:
Yeap no worries